docker-config/secrets.json

```
@@ -0,0 +1,11 @@
+{
+  "DB": {
+    "SECRET": "Very hidden secret"
+  },
+  "KEYCLOAK": {
+    "REALM_NAME": "SPSH",
+    "CLIENT_ID": "spsh",
+    "SECRET": "w0clVkJCNyY9IseOMikCduEYG1zdodz6",
+    "ADMIN_SECRET": "w0clVkJCNyY9IseOMikCduEYG1zdodz6"
+  }
+}
```

Consistent file ending
```suggestion
}

```